src/main/java/com/dev/cinema/util/HibernateUtil.java

```diff
@@ -0,0 +1,23 @@
+package com.dev.cinema.util;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+    private static SessionFactory sessionFactory = initSessionFactory();
+
+    public HibernateUtil() {
+    }
+
+    private static SessionFactory initSessionFactory() {
+        try {
+            return new Configuration().configure().buildSessionFactory();
+        } catch (Exception e) {
+            throw new RuntimeException("Cant creating session factory", e);
```

Comment:
```suggestion
            throw new RuntimeException("Cant creating session factory", e);
```
here the same situation with custom exception